shells/electron/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "main": "./index.js",
+  "scripts": {
+    "build": "webpack --config webpack.backend.js && webpack --config webpack.config.js"
```

Comment:
yeah. What you changed it to was what I had to run locally to reuse that webpack. :thumbsup: